src/app/util/generic-callback-stubs.cpp

```diff
@@ -34,18 +34,18 @@ bool __attribute__((weak)) emberAfAttributeWriteAccessCallback(EndpointId endpoi
     return true;
 }
 
-EmberAfStatus __attribute__((weak))
+Status __attribute__((weak))
```

Comment:
Where does this file declare `Status` as a bareword?

This is why we should not have `using` in headers.

Reply:
fixed in PR https://github.com/project-chip/connectedhomeip/pull/32055